tests/queries/0_stateless/01037_polygon_dicts_correctness_all.sh

```diff
@@ -34,7 +34,7 @@ CREATE TABLE test_01037.polygons_array
 ENGINE = Memory;
 "
 
-$CLICKHOUSE_CLIENT --query="INSERT INTO test_01037.polygons_array FORMAT JSONEachRow" --max_insert_block_size=100000 < "${CURDIR}/01037_polygon_data"
+$CLICKHOUSE_CLIENT --query="INSERT INTO test_01037.polygons_array FORMAT JSONEachRow" --min_chunk_bytes_for_parallel_parsing=10485760 --max_insert_block_size=100000 < "${CURDIR}/01037_polygon_data"
```

Comment:
But the fact we cannot parse very long records with parallel parsing is alarming.